src/D2662.tex

```diff
@@ -19,108 +19,117 @@
 that can work in all contexts.
 \item We want a syntax that can be expanded to support slicing in the future.
-\item Pack elements can be array-like or tuple-like, so we need to be careful
-  about ambiguities.In particular, directly applying a subscript to a pack
-  (\tcode{P[i]}, where \tcode{P} is a pack) is not viable. Indeed, indexing a
+\item Pack elements can be array like or tuple like, so we need to be careful
+  about ambiguities. In particular, directly applying a subscript to a pack
+  (\tcode{P[i]}, where \tcode{P} is a pack) is nonviable. Indeed, indexing a
   pack of arrays (\tcode{ArrayPack...[index]}), indexing each array in a pack
-  (\tcode{ArrayPack[index]...}), indexing a single array with a pack of indexes
+  (\tcode{ArrayPack[index]...}), indexing a single array with a pack of indices % lah: indexes is for book indexes and indices is for mathematics.  
   (\tcode{Array[IndexPack]...}), and indexing each array in a pack of arrays
-  with an index from an (equal-length) pack of indexes
+  with an index from an (equal-length) pack of indices 
   (\tcode{ArrayPack[IndexPack]...}) are different operations, all of which are
   useful. [\href{https://compiler-explorer.com/z/E86h8eMG1}{This example}]
   shows how, using the proposed syntax, one can distinguish between indexing
-  the arrays from a pack with indexes from a pack
+  the arrays from a pack with indices from a pack
   (\tcode{ArrayPack[IndexPack]...}) and indexing a pack of arrays with a pack
-  of indexes (\tcode{ArrayPack...[IndexPack]...}).
+  of indices (\tcode{ArrayPack...[IndexPack]...}).
 \end{itemize}
 
 \subsubsection{pack.[index]}
 
 Historically in C++ (and C++-like languages), a single dot denotes member access.
-Reusing that syntax for pack indexing would introduce a semantics inconsistency.
+Reusing that syntax for pack indexing would introduce a semantics inconsistency and, more importantly, could close the door to future evolutions.
 
-More importantly, it may close the door to future evolutions.
-\paper{P1858R0} proposes \tcode{tuple-like.[N]} as sugar over \tcode{get<N>(tuple-like)}
-and \tcode{aggregate.[N]} returning the $Nth$ data member of an aggregate.
+\paper{P1858R0} proposes \tcode{tuple-like.[N]} as sugar over % lah: What do you mean "as sugar over"? If I'm not clear, then a global audience won't be clear. 
+\tcode{get<N>(tuple-like)}
+and \tcode{aggregate.[N]} returning the $Nth$ data member of an aggregate. % lah: WHAT is "returning the $Nth$ data member of an aggregate"---just the aggregate.[N] or BOTH get<N>(tuple-like) AND aggregate.[N]? 
 
-Ideally, \tcode{tuple[N]} would just work. A few proposals have tried to improve the ergonomy of tuple indexing
-\paper{P2726R0}, \paper{P0311R0}.
+Ideally, \tcode{tuple[N]} would just work. % lah: By "just," do you mean that it would simply work or do you mean that it would barely work? 
+A few proposals have tried to improve the ergonomy % lah: Do you mean ergodicity? 
```

Comment:
user-friendliness is probably what we want to say.

Reply:
Okay, so change that to user-friendliness